noxfile.py

```
@@ -0,0 +1,74 @@
+"""Nox automation file for cpp-linter project.
+
+This file defines automation sessions for testing, coverage, and documentation
+using uv for dependency management and virtual environment backend.
+"""
+
+import logging
+from os import environ
+import sys
+import nox
```

Pylint (3.3.7)

[error] Unable to import 'nox'

(E0401)